datadog-ffe-ffi/Cargo.toml

```
@@ -0,0 +1,26 @@
+# Copyright 2025-Present Datadog, Inc. https://www.datadoghq.com/
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "datadog-ffe-ffi"
+edition.workspace = true
+version = "0.1.0"
+rust-version.workspace = true
+license.workspace = true
+
+[lib]
+crate-type = ["lib", "staticlib", "cdylib"]
+bench = false
+
+[features]
+default = ["cbindgen"]
+cbindgen = ["build_common/cbindgen", "ddcommon-ffi/cbindgen"]
+
+[build-dependencies]
+build_common = { path = "../build-common" }
+
+[dependencies]
+datadog-ffe = { path = "../datadog-ffe" }
```

This needs to be pinned to exact version:
```suggestion
datadog-ffe = { path = "../datadog-ffe", version = "=0.1.0" }
```